### PR TITLE
Provides an tester for Lagom-play-scala.

### LINKED ISSUE #243: Play sample app should interact with Lagom
When starting a Play app next to a Lagom service they should be connected to expand the test surface.

See ` sbt-conductr-tester` sample apps.

### PR DESCRIPTION
**Pull request by ignasi35**
Fixes #243 ( java was complete, this PR provides scala).

Moves existing `lagom-bundle` to `lagom-java-bundle` and adds `lagom-scala-bundle`.

This tester provides a lagom service and a play app acting as a client. When installed you should be able to:

```
$  curl http://192.168.10.1:9000/foo
hardcoded-foo-scala-response

$  curl http://192.168.10.1:9000/
Hello world

$  curl http://192.168.10.1:9000/lagom-redirect
via-play hardcoded-foo-scala-response
```

Merging this PR will complete the changes originally introduced in https://github.com/typesafehub/sbt-conductr/pull/223 so #223 can be removed.

This PR can't be backported to `2.2.x` because it relies on Lagom's Scala API not available in Lagom `1.2.x` (which is used in `sbt-conductr/2.2.x`).

### CODE REVIEW

File: sbt-conductr-tester/lagom-scala-bundle/project/plugins.sbt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.3.1")
+
+//lazy val root = Project("plugins", file(".")).dependsOn(plugin)
```
Comment (markusjura) [inline sbt-conductr-tester/lagom-scala-bundle/project/plugins.sbt]: Uncomment these lines to depend on the working version

**Comment by ignasi35** [comment]
This PR needs squashing before merging.

File: sbt-conductr-tester/lagom-scala-bundle/play-service/app/PlayGatewayLoader.scala [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  lazy val itemController = wire[MyController]
+}
+
+class PlayGatewayLoader extends ApplicationLoader {
+  override def load(context: Context) = context.environment.mode match {
+    case Mode.Dev =>
+      new PlayGateway(context) with LagomDevModeComponents {}.application
+    case _ =>
+      new PlayGateway(context) with ConductRServiceLocatorComponents with ConductRLifecycleComponents {}.application
+  }
```
Comment (ignasi35) [inline sbt-conductr-tester/lagom-scala-bundle/play-service/app/PlayGatewayLoader.scala]: Actually, I'm not sure `ConductRLifecycleComponents` is necessary.

**Comment by markusjura** [comment]
jenkins

**Pull request merged**